processes/app_release.md

```diff
@@ -44,5 +47,6 @@
 15. Open the app's main repo and close the vx.y.z milestone (Issues -> Milestones in the Right Upper Corner)
 
 16. Create a milestone for the next patch version
-
-17. If the release was for patch version, forward-port the changelog to master/main (`git checkout stablex.x CHANGELOG.md`)
+17. If you are releasing a minor or major release, you need to branch off this minor or major release: `git checkout -b  stablex.x, push origin stablex.x`
```

Comment:
doesn't say where you branch off. this needs to be the target branch *after* the release branch has been merged and the local branch updated.

and split the two commands, now it looks like the checkout and push can be run at once.